.pre-commit-config.yaml

```diff
@@ -19,7 +19,7 @@ repos:
         language: system
       - id: pylint
         name: pylint
-        entry: 'pylint --rcfile=.github/scripts/podio.pylint.py3k.rc --py3k --msg-template="{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}"'
+        entry: 'pylint --py3k --msg-template="{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}"'
```

Comment:
`--py3k` disables all the "normal" pylint warnings, and it would be good to have a .rc file someline along the lines of [this](https://github.com/AIDASoft/DD4hep/blob/master/.github/scripts/DD4hep.pylint.rc)

Reply:
I see. I have never actually looked to closely into the pylint call to be honest. However, simply removing the `--py3k` already makes pylint complain a lot more ;) On the other hand the configuration from DD4hep you linked seems to be incompatible with the default `flake8` options, mainly related to formatting from a quick glance. So we will have to at least find a consistent configuration for both.

Reply:
yes, you need also the `setup.cfg` e.g. https://github.com/AIDASoft/DD4hep/blob/master/setup.cfg

Reply:
With the right configuration (taken from ilcdirac) and fixing the things pylint complains about everything seems to fine.